openshift/Dockerfile.openshift

```diff
@@ -4,15 +4,16 @@ WORKDIR /build
 COPY . .
 RUN GO111MODULE=on CGO_ENABLED=0 GOOS=${GOOS} GOPROXY=${GOPROXY} go build \
   -ldflags="-w -s -X 'main.version=${VERSION}'" \
-  -o=cluster-api-provider-azure-controller-manager \
+  -o=manager \
```

Comment:
If you change this (which I'm happy with) we should also change the one over in the capiinstaller here: https://github.com/openshift/cluster-capi-operator/blob/3f3e845f5daf0d1fc70a188117152660e5de18c4/pkg/controllers/capiinstaller/component_customizer.go#L41

Reply:
Good catch. Changed in the cluster-capi-operator PR.